python/setup.py

```diff
@@ -0,0 +1,78 @@
+""""
+This build script is modeled after the pyspark package in the apache/spark
+repository.
+
+https://github.com/apache/spark/blob/master/python/setup.py
+"""
+
+from setuptools import setup
+import os
+import glob
+import sys
+import shutil
+
+
+VERSION = '2.1.0'
```

Comment:
It doesn't look like this matches the version in `build.sbt`. Do they need to be managed together? Has this deploy run successfully?

Reply:
The versions technically don't need to be managed together, but any update to the scala library should probably retrigger a new python package. I've uploaded this successfully to pypi testing and one annoying thing is that you can't reupload a package once it's been uploaded.

Reply:
Yeah, I've run into that before as well. My question is, how does `JAR_FILE = "*-assembly-{}-*.jar".format(VERSION)` match the correct jar if the versions don't match?

Reply:
Actually yes, you're right, they do have to be managed together otherwise the packaging fails. I ran into it when I bumped the version number because of pypi's weird limitations. I might try reading the version from a file in the root directory.